app/src/main/java/com/github/gogetters/letsgo/database/Authentication.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.gogetters.letsgo.database
+
+import com.google.firebase.auth.FirebaseUser
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+
+class Authentication {
```

Comment:
Is it not better to make it implement an interface so we can mock it? Maybe too late thought

Reply:
I believe sticking to Emulating everything is better